packages/manager/modules/pci/src/projects/project/storages/containers/add/add.html

```diff
@@ -0,0 +1,70 @@
+<div class="p-5">
+    <oui-back-button
+        data-on-click="$ctrl.goBack()">{{:: 'pci_projects_project_storages_containers_add_back_label' | translate }}</oui-back-button>
+
+    <h1 data-translate="pci_projects_project_storages_containers_add_title"></h1>
+
+    <cui-message-container data-messages="$ctrl.messages"></cui-message-container>
+
+    <div>
+        <oui-stepper
+            data-on-finish="$ctrl.add()">
+            <oui-step-form
+                data-header="{{:: 'pci_projects_project_storages_containers_add_region_title' | translate }}"
+                data-loading="$ctrl.loadings.regions"
+                data-navigation="$ctrl.container.region"
+                data-on-focus="$ctrl.onRegionsFocus()"
+                data-on-submit="$ctrl.onRegionChange()"
+                data-editable="!$ctrl.loadings.save">
+                <pci-project-regions-list
+                    data-regions="$ctrl.regions"
+                    data-selected-region="$ctrl.container.region"
+                    data-display-selected-region="$ctrl.displaySelectedRegion"></pci-project-regions-list>
+            </oui-step-form>
+
+            <oui-step-form
+                data-header="{{:: 'pci_projects_project_storages_containers_add_type_title' | translate }}"
+                data-ng-if="!$ctrl.archive"
+                data-position="2"
+                data-on-focus="$ctrl.onTypesFocus()"
+                data-on-submit="$ctrl.onTypeChange()"
+                data-navigation="$ctrl.selectedType"
+                data-editable="!$ctrl.loadings.save">
+                <div class="container-fluid px-0">
```

Comment:
`.container-fluid` already apply paddings so removing them by adding `px-0` sounds weird.

Is this element still needed?

Reply:
We don't want to apply default margin each time.

So, I think it's needed :)